dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -301,22 +302,26 @@ public FunctionView Describe()
     /// <inheritdoc/>
     public Task<SKContext> InvokeAsync(
         string? input = null,
+        CancellationToken cancellationToken = default,
```

Comment:
Same note about CT typically coming last.  It's more of an issue when there aren't so many optional parameters, but even so it's relevant.  Devs are used to seeing it last in docs, and there's tooling that expects it to be last, e.g. analyzers that will look at the last parameter to see whether it's a CT and act accordingly.